.circleci/config.yml

```diff
@@ -27,12 +27,13 @@ jobs:
           git checkout $CIRCLE_BRANCH || git checkout master
           export WEBAPP_GIT_COMMIT=$(git rev-parse HEAD)
           echo "$WEBAPP_GIT_COMMIT"
-          export CURL_FAILED=0
-          curl -f -o ./dist.tar.gz https://pr-builds.mattermost.com/mattermost-webapp/commit/${WEBAPP_GIT_COMMIT}/mattermost-webapp.tar.gz && mkdir ./dist && tar -xvf ./dist.tar.gz -C ./dist --strip-components=1 || export CURL_FAILED=1
 
-          if [ $CURL_FAILED -eq 1 ]
+          curl -f -o ./dist.tar.gz https://pr-builds.mattermost.com/mattermost-webapp/commit/${WEBAPP_GIT_COMMIT}/mattermost-webapp.tar.gz
+          if [[ $? == 0 ]]
           then
-            npm ci && cd node_modules/mattermost-redux && npm install && npm run build && cd ../.. && make build
```

Comment:
Is `cd node_modules/mattermost-redux && npm install && npm run build` no longer needed?

Reply:
That's where I am actually not so sure. But since mattermost-redux is specified in package.json already with the specific commit https://github.com/mattermost/mattermost-webapp/blob/5a814d9ab40b0a35b4e3f03a53b2e473aaefde68/package.json#L41 I do not see a reason why it is needed still.

Reply:
Hm, you're right. I wonder why it was there in the first place.